template/runtime/src/lib.rs

```diff
@@ -270,7 +270,7 @@ pub struct FixedGasPrice;
 impl FeeCalculator for FixedGasPrice {
 	fn min_gas_price() -> U256 {
 		// Gas price is always one token per gas.
```

Comment:
Don't forget the comment.

Maybe the dynamic fee should just implement the FeeCalculator trait directly.

Reply:
Thanks for your review. Done!